src/elements/content-preview/PreviewMask.tsx

```diff
@@ -17,7 +17,7 @@ export default function PreviewMask({ errorCode, extension, isLoading }: Props):
 
     return (
         <div className="bcpr-PreviewMask">
-            {errorCode ? <PreviewError /> : isLoading && <PreviewLoading extension={extension} />}
+            {errorCode ? <PreviewError errorCode={errorCode} /> : isLoading && <PreviewLoading extension={extension} />}
         </div>
```

Comment:
Can we update test for this ?

Reply:
I believe updating the snapshot is sufficient, since there's already two tests for preview error, one of which deals with the 'forbidden_by_policy' error:
```
test('should render correctly', () => {
            const wrapper = getWrapper();
            expect(wrapper).toMatchSnapshot();
        });

        test('should render correctly when blocked by access policy', () => {
            const wrapper = getWrapper({ errorCode: ERROR_CODE_FETCH_FILE_DUE_TO_POLICY });
            expect(wrapper).toMatchSnapshot();
        });
```